distbench_utils.cc

```diff
@@ -743,10 +743,32 @@ GetCanonicalDistributedSystemDescription(
   return ret;
 }
 
+absl::Status ValidateTestsSetting(const TestsSetting& settings) {
+  if (!settings.has_keep_instance_log()) {
+    return absl::InvalidArgumentError(
+        "TestsSettings does not have an option for keep_instance_log");
+  }
+  if (!settings.has_shutdown_after_tests()) {
+    return absl::InvalidArgumentError(
+        "TestsSettings does not have an option for shutdown_after_tests");
+  }
+
+  return absl::OkStatus();
+}
+
 absl::StatusOr<TestSequence> GetCanonicalTestSequence(
     const TestSequence& sequence) {
   TestSequence ret;
-  *ret.mutable_tests_setting() = sequence.tests_setting();
+  if (sequence.has_tests_setting()) {
+    auto status = ValidateTestsSetting(sequence.tests_setting());
+    if (!status.ok()) {
+      return status;
+    }
+    *ret.mutable_tests_setting() = sequence.tests_setting();
+  } else {
+    return absl::InvalidArgumentError(
+        "Test Sequence doesn't have tests setting");
```

Comment:
this is fine....

Reply:
is it fine to not have tests settings, or is the code fine?